package.json

```diff
@@ -66,7 +66,7 @@
   "peerDependencies": {
     "@types/react": "*",
     "@types/react-native": "*",
-    "@discordapp/flash-list": "1.0.0",
+    "@shopify/flash-list": "https://github.com/discord/flash-list/archive/91d56d0fdfbf08d670732dafe328f8f2664416dc.tar.gz",
```

Comment:
let's depend on the Shopify's official version (`^1.6.1`) so this would be more likely to be accepted as a PR to the upstream. Maybe `*` would be fine here too